.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": "0.2.0",
```

Comment:
This is superbench's version? Looks we have many places with superbench version.

Reply:
no, https://code.visualstudio.com/docs/editor/debugging#_global-launch-configuration